lib/transformers/allOf.js

```diff
@@ -80,15 +74,57 @@ function allOf({schema, options, transform, data, schemaErrors} = {}) {
         });
 
         return acc;
+    }
+    if ('allOf' in sectionSchema) {
+        return sectionSchema.allOf.reduce((acc, propertiesSchema) => {
+            return transformPropertySchema(acc, propertiesSchema, acc);
+        }, {});
+    }
+    const acc = {};
+    const propertiesSchema = Object.fromEntries(
+        Object.entries(sectionSchema).filter(([key]) => key === 'properties')
+    );
+    return transformPropertySchema(acc, propertiesSchema);
```

Comment:
remove `const acc = {};` as its only purpose is to be passed in to the function in the return statement (use an object literal instead `{}`).

as discussed, something like:

```js
    const schemaProperties = sectionSchema.properties;
    return transformPropertySchema({}, schemaProperties);
}
```
I think `schemaProperties` is a better name of the variable here also as it is the properties of the schema being accessed here.